package.json

```diff
@@ -37,7 +37,7 @@
     "kill:port": "kill -s TERM $(lsof -t -i:8080 -sTCP:LISTEN) || true"
   },
   "engines": {
-    "node": "^12.19.0 || ^14.15.0 || ^16.13.0 || ^18.14.0 || ^20.5.0"
+    "node": "^12.19.0 || ^14.15.0 || ^16.13.0 || ^18.14.0 || <=20.5.0"
```

Comment:
Is this supposed to be `<=22.5.1`?

Reply:
My intention is to include all versions greater than `20.5.0` (which would include 22.x)

Reply:
I see, then this should be `>=20.5.0`

Reply:
https://docs.npmjs.com/cli/v11/configuring-npm/package-json#engines
I confused myself. Although I did do a test installation and got no errors 🤔

Reply:
I definitely hit the `=` condition when I tested the installation locally 🤦